data/migrations/20190110100917_shopping_list.js

```diff
@@ -10,6 +10,12 @@ exports.up = function(knex, Promise) {
       table.foreign('recipe_id').references('id').on('recipe');
       table.string('recipe_name').unsigned().unique();
       table.foreign('recipe_name').references('recipe_name').on('recipe');
+      table.string('ingredient').unsigned();
+      table.foreign('ingredient').references('ingredient').on('ingredients');
+      table.integer('quantity_to_buy');
```

Comment:
I think that this is meant to be a float (so that it can have decimal numbers stored in it)! It could also be argued that it shouldn't be nullable. If we were using something else other than sqlite, we could use `defaultTo`.
```javascript
table.float('quantity_to_buy').notNullable();
```
or, for production:
```javascript
table.float('quantity_to_buy').defaultTo(1);
```